traveler-swift-core/TravelerKit/Delegates/WishlistAddDelegate.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  WishlistAddDelegate.swift
+//  TravelerKit
+//
+//  Created by Omar Padierna on 2019-09-03.
+//  Copyright © 2019 Ata Namvari. All rights reserved.
+//
+
+import Foundation
+
+/// Notified when wishlisting item
+public protocol WishlistAddDelegate: class {
```

Comment:
Naming is a little weird here...

`WishlistAdditionDelegate`....nouns instead of verbs